vm/src/stdlib/sys.rs

```diff
@@ -590,7 +590,12 @@ mod sys {
     fn unraisablehook(unraisable: UnraisableHookArgs, vm: &VirtualMachine) {
         if let Err(e) = _unraisablehook(unraisable, vm) {
             let stderr = super::PyStderr(vm);
-            writeln!(stderr, "{}", e.as_object().repr(vm).unwrap().as_str());
+            let default = vm.ctx.intern_str("").to_owned();
+            writeln!(
+                stderr,
+                "{}",
+                e.as_object().repr(vm).unwrap_or(default).as_str()
```

Comment:
```suggestion
                e.as_object().repr(vm).unwrap_or_else(|| vm.ctx.empty_str.to_owned()).as_str()
```
We have a shared empty_str

Reply:
Fixed